ykrt/src/mt.rs

```diff
@@ -163,10 +163,8 @@ impl MT {
             TransitionLocation::Execute(ctr) => {
                 #[cfg(feature = "yk_jitstate_debug")]
                 print_jit_state("enter-jit-code");
-                let ptr = ctr.exec(ctrlp_vars, frameaddr);
-                #[cfg(feature = "yk_jitstate_debug")]
-                print_jit_state("exit-jit-code");
-                return ptr;
+                ctr.exec(ctrlp_vars, frameaddr);
```

Comment:
Did you ask Laurie if the `Arc` remains consistent when you rug-pull Rusts's stack?

Reply:
Yeah, see main conversation.